x11-terms/kitty/files/kitty-9999-flags.patch

```diff
@@ -1,7 +1,7 @@
 Respect user CFLAGS.
```

Comment:
... and delete this file and make it use the previously introduced patch. In future, you don't have to bump patches for every version if its the same. You can use the previous one if it still works.